vsts-fullbuild-redhat-vm/azuredeploy.json

```diff
@@ -0,0 +1,205 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vstsAccountURL": {
+      "type": "string",
+      "metadata": {
+        "description": "Team Services Account URL (e.g. https://myaccount.visualstudio.com)"
+      }
+    },
+    "vstsPAT": {
+      "type": "string",
+      "metadata": {
+        "description": "Team Services PAT for user with Agent Pools (read,manage) permissions"
+      }
+    },
+    "vstsPoolName": {
+      "type": "string",
+      "defaultValue": "Default",
+      "metadata": {
+        "description": "Team Services Agent Pool Name"
+      }
+    },
+    "vstsAgentName": {
+      "type": "string",
+      "defaultValue": "redhat-build-full",
+      "metadata": {
+        "description": "Team Services Agent Name"
+      }
+    },    
+    "adminUserName": {
+      "type": "string",
+      "defaultValue": "vstsbuild",
+      "metadata": {
+        "description": "Linux VM User Account Name"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "dnsLabelPrefix": {
```

Comment:
We could generate this and I agree the name is not technically important; however, our team uses standard naming conventions to keep up with the purpose of our VMs and therefore I assume other teams may want to do this as well, so giving the user the control in naming makes the most sense to me.